app/views/miq_action/show.html.haml

```diff
@@ -191,7 +191,7 @@
         %label.control-label.col-md-2
           = _("Playbook Catalog Item")
         .col-md-8
-          = h(ServiceTemplateAnsiblePlaybook.order(:name).pluck(:name, :id).find { |(_, id)| id == @record.options[:service_template_id] }[0])
+          = h(@record.options[:service_template_name].find { |(_, id)| id == @record.options[:service_template_id] }[0])
```

Comment:
it is odd to store 2 pieces of data in a key called `:service_template_name` and only use the latter part, which is an id (assuming a different one than in `@record.options[:service_template_id]`)

Reply:
I changed how `service_template_name` is stored in ruby with only name. And in haml I only need to call `@record.options[:service_template_name]` I am wondering whether if this works. Thank you!